lib/SILOptimizer/Mandatory/TFDeabstraction.cpp

```diff
@@ -2043,7 +2043,27 @@ void TFDeabstraction::formGraphOp(SILTensorOpInfo &opInfo,
     switch (operandClass.second) {
     case SILTensorOpInfo::OperandClass::Input:
     case SILTensorOpInfo::OperandClass::InputElt:
+      assert(0 && "Input classes cannot exist for attributes");
     case SILTensorOpInfo::OperandClass::Normal:  // No modifier.
+      switch (constValue.getKind()) {
```

Comment:
Can you refactor this into a function or lambda?

Reply:
done